### ISSUE TITLE
The question of training loss

### ISSUE THREAD

**Issue by 18483666678** [YunYang1994/tensorflow-yolov3#258]
Training data set or voc20 data set, training loss and test loss are both greater than zero.  I used the following configuration:
# Train options
__C.TRAIN                       = edict()
__C.TRAIN.ANNOT_PATH            = "./data/dataset/voc_train.txt"
__C.TRAIN.BATCH_SIZE            = 2
__C.TRAIN.INPUT_SIZE            = [320, 352, 384, 416, 448, 480, 512, 544, 576, 608]
__C.TRAIN.DATA_AUG              = True
__C.TRAIN.LEARN_RATE_INIT       = 1e-4
__C.TRAIN.LEARN_RATE_END        = 1e-6
__C.TRAIN.WARMUP_EPOCHS         = 2
__C.TRAIN.FISRT_STAGE_EPOCHS    = 20
__C.TRAIN.SECOND_STAGE_EPOCHS   = 30
__C.TRAIN.INITIAL_WEIGHT        = "./checkpoint\yolov3_coco_demo.ckpt"
The training results are as follows:
=> Epoch: 65 Time: 2019-07-26 01:23:30 Train loss: 1.43 Test loss: 6.71 Saving ./checkpoint/yolov3_test_loss=6.7142.ckpt ...
train loss: 0.81: 100%|██████████| 1857/1857 [07:30<00:00,  4.04it/s]
=> Epoch: 66 Time: 2019-07-26 01:32:07 Train loss: 1.46 Test loss: 6.35 Saving ./checkpoint/yolov3_test_loss=6.3457.ckpt ...
train loss: 1.19: 100%|██████████| 1857/1857 [08:08<00:00,  4.04it/s]
=> Epoch: 67 Time: 2019-07-26 01:41:22 Train loss: 1.41 Test loss: 6.68 Saving ./checkpoint/yolov3_test_loss=6.6832.ckpt ...
train loss: 1.24: 100%|██████████| 1857/1857 [07:51<00:00,  4.01it/s]
=> Epoch: 68 Time: 2019-07-26 01:50:18 Train loss: 1.39 Test loss: 6.48 Saving ./checkpoint/yolov3_test_loss=6.4826.ckpt ...
train loss: 1.12: 100%|██████████| 1857/1857 [07:34<00:00,  4.49it/s]
=> Epoch: 69 Time: 2019-07-26 01:58:59 Train loss: 1.41 Test loss: 6.77 Saving ./checkpoint/yolov3_test_loss=6.7692.ckpt ...
train loss: 0.96: 100%|██████████| 1857/1857 [07:24<00:00,  3.67it/s]
=> Epoch: 70 Time: 2019-07-26 02:07:28 Train loss: 1.38 Test loss: 6.70 Saving ./checkpoint/yolov3_test_loss=6.6956.ckpt ...
Training batch 90, the loss is still not reduced, is this normal?

**Comment by YunYang1994** [comment]
Be patient， just wait until it's done !
